api/telemetry/v1alpha1/otlp_config.go

```diff
@@ -65,7 +79,18 @@ type BackOffConfig struct {
 
 	// MaxElapsedTime is the maximum amount of time (including retries) spent trying to send a request/batch.
 	// Once this value is reached, the data is discarded. If set to 0, the retries are never stopped.
-	MaxElapsedTime time.Duration `json:"max_elapsed_time,omitempty"`
+	MaxElapsedTime *time.Duration `json:"max_elapsed_time,omitempty"`
+}
+
+func (b *BackOffConfig) SetDefaultBackOffConfig() {
+	if b == nil {
+		b = &BackOffConfig{}
+	}
+
+	b.Enabled = true
+	if b.MaxElapsedTime == nil {
+		b.MaxElapsedTime = utils.ToPtr(0 * time.Second)
```

Comment:
Please mention this default value in the `MaxElapsedTime` doc comment because the retries are never stopped by default (which might be surprising to folks who are used to the opposite behavior in the OtelCol).